### DISCUSSION TITLE
Cyberduck ftp host configuration

### DISCUSSION THREAD

**Discussion by RobertSeymour1** [iterate-ch/cyberduck#16020] [Q&A]
I am having host  login issues.
What is the meaning of the gold warning triangle right of the URL? (See attached)
Is something wrong with the URL?

Thanks
Robert
<img width="591" alt="Screenshot 2024-06-12 at 8 32 25 AM" src="https://github.com/iterate-ch/cyberduck/assets/54649337/6d8b3ddf-79af-4ca1-b340-ae4733d1de17">

**Comment by tatze96** [comment]
The warning triangle beside the URL indicates that the current URL is unreachable. Please verify that you've entered the correct information for your server. If you are unsure which information you need to enter in the respective fields to connect to the server successfully, contact your server administrator.

**Comment by RobertSeymour1** [comment]
Thanks.

Robert
